packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -132,30 +132,36 @@ depth = model.infer_image(raw_img) # HxW raw depth map in numpy
 const diffusers_default = (model: ModelData) => [
 	`from diffusers import DiffusionPipeline
 
-pipeline = DiffusionPipeline.from_pretrained("${model.id}")`,
+pipe = DiffusionPipeline.from_pretrained("${model.id}")
+
+prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
+image = pipe(prompt).images[0]`,
```

Comment:
Nit: as discussed in Slack, having a prompt might be a bit confusing due to many models expecting certain tokens. In any case, this is fine I think to showcase usage